sendgrid/helpers/mail/validators.py

```diff
@@ -0,0 +1,68 @@
+################################################################
+# Various types of Validators
+################################################################
+
+class ValidateAPIKey(object):
+    """Validates content to ensure SendGrid API key is not present"""
+
+    regexes = None
+
+    def __init__(self, regex_strings=list(), use_default=True):
+        """Constructor
+        Args:
+            regex_strings (list<str>): list of regex strings
+            use_default (bool): Whether or not to include default regex
+        """
+
+        import re
+        self.regexes = set()
+
+        #Compile the regex strings into patterns, add them to our set
+        for regex_string in regex_strings:
+            self.regexes.add(re.compile(regex_string))
+
+        if use_default:
+            default_regex_string = 'SG\.[0-9a-zA-Z]+\.[0-9a-zA-Z]+'
+            self.regexes.add(re.compile(default_regex_string))
+
+
+    def validate_message_dict(self, request_body):
+        """With the JSON dict that will be sent to SendGrid's API, 
+            check the content for SendGrid API keys - throw exception if found
+        Args:
+            request_body (:obj:`dict`): message parameter that is
+                                            an argument to: mail.send.post()
+        Raises:
+            APIKeyIncludedException: If any content in request_body matches regex
+        """
+
+        #Handle string in edge-case
```

Comment:
PEP8 recommends to have a space after the `#` character.

> An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.

* http://pep8.org/#inline-comments